presto-hive/src/main/java/com/facebook/presto/hive/rcfile/RcFilePageSource.java

```diff
@@ -105,12 +105,6 @@ public RcFilePageSource(
     }
 
     @Override
-    public long getTotalBytes()
-    {
-        return rcFileReader.getLength();
```

Comment:
With this change `RcFileReader::getLength()` can be removed too as it is not used by anywhere else.